.github/workflows/smoke.yml

```diff
@@ -51,9 +51,7 @@ jobs:
       # Download the Proxy cache. The job is ideally 100% cached so no real calls are made.
       - name: Download cache
         run: |
-          mkdir test-cache
-          cd test-cache
-          gh run download --repo dependabot/smoke-tests --name cache-${{ matrix.suite }}
```

Comment:
This crept in via a copy paste from `core`/`proxy`... they don't set the `--dir` name arg. I'll open PR's to fix those.